gtsfm/bundle/bundle_adjustment.py

```diff
@@ -9,13 +9,15 @@
 import dask
 import gtsam
 from dask.delayed import Delayed
-from gtsam import GeneralSFMFactorCal3Bundler, SfmData, symbol_shorthand
+from gtsam import GeneralSFMFactor2Cal3Bundler, SfmData, symbol_shorthand
 
 from gtsfm.common.sfm_result import SfmResult
 
 # TODO: any way this goes away?
-C = symbol_shorthand.C
-P = symbol_shorthand.P
+C = symbol_shorthand.C  # camera
+P = symbol_shorthand.P  # 3d point
+X = symbol_shorthand.X  # camera pose
+K = symbol_shorthand.K  # calibration
```

Comment:
I would like to include the info still, to help users understand why we use the GeneralSfmFactor2 instead of GeneralSfmFactor (and maybe point to URL documentation of each -- and how factors differe mathematically)

Reply:
added a small note on why we shift. Mathematically, the factors are the same.